node/snippets/documents/set-document.ts

```diff
@@ -0,0 +1,27 @@
+import { grpc } from '@nitric/sdk';
+
+// [START import]
+import { documents } from '@nitric/sdk';
+// [END import]
+
+const proto = grpc.document.DocumentServiceClient.prototype;
+
+test('Should set a documents content', async () => {
+  jest.spyOn(proto, 'set').mockImplementation((_, cb: any) => cb(null, true));
+
+  // [START snippet]
+  interface Product {
+    id: string;
+    name: string;
+    description: string;
+  }
+
+  const collection = documents().collection<Product>('products');
+
+  await collection.doc('nitric').set({
+    id: 'nitric',
+    name: 'nitric',
+    description: 'A development framework!',
+  });
+  // [END snippet]
+});
```

Comment:
Is it possible to get this to work for the individual SDK repos, my only concern here is that this repo will need to have alot of tooling to support all of our SDKs, where our existing SDKs already have this tooling on their own and are capable of running tests/building examples etc.

Reply:
Good idea, we could clone the repos and perform the glob matching on them. Or potentially use the github contents API.